client/src/index.css

```diff
@@ -327,11 +327,10 @@ button.home-tile.btn {
   font-weight: 900;
 }
 
-.home-tile:hover {
+.home-tile-col:hover {
   background: #dae3ee;
 }
-.home-tile:hover,
```

Comment:
Why remove this line?

Reply:
Background color on hover effect does not fill the whole box.